### DISCUSSION TITLE
How do I get rid of my locally built Rectangle?

### DISCUSSION THREAD

**Discussion by slifty** [rxhanson/Rectangle#712] [Q&A] [answered]
This is a pretty niche question -- probably only applies to people who contribute -- but my locally built version of rectangle seems to be superseding the version in my Applications directory for things like starting-on-boot.

Do you happen to know what I should do to blow away the debug artifacts of my local build?

**Comment by rxhanson** [comment, marked as answer]
Ah, sorry I forgot to reply to you on this one.

You can clean the build folder (Product -> Clean Build Folder or Shift+cmd+K). 

You can also clear out all the temporary xcode stuff, which is sometimes useful if Xcode is acting weird:
https://stackoverflow.com/questions/38016143/how-can-i-delete-derived-data-in-xcode-8

Let me know if that doesn't get it done for you.
